web-components/src/components/views/views.stories.tsx

```diff
@@ -0,0 +1,10 @@
+import ErrorView from './ErrorView';
+import NotFound from './NotFoundView';
+
+export default {
+  title: 'Views',
+  component: NotFound,
+};
+
+export const notFoundView = (): JSX.Element => <NotFound />;
+export const errorView = (): JSX.Element => <ErrorView />;
```

Comment:
Optional - It might be nice to have the bee image here. That will help future devs to identify that this component is built and in our component library.

Reply:
yes, you are right but 'm leaving it as it is for the moment since I have some doubts because we mainly have the assets in the registry, etc.